frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ClusterDetails/ClusterDetails.tsx

```diff
@@ -79,6 +84,7 @@ export default function ClusterDetailsPage({ match }: RouteComponentProps<{ id:
             clusterManagementAddonsState,
             clusterClaimsState,
             clusterCuratorsState,
+            ansibleJobState,
```

Comment:
Why not just read the ansibleJobs from the recoil cache in the DistributionInfo component instead of doing all the set up and context setting and prop passing down in all the other files?